bootstrap.py

```diff
@@ -74,14 +72,39 @@
 
 # ---- Install sub repos
 
+from install_dev_repos import REPOS, install_repo
+from spyder import get_versions
+from spyder.config.base import get_conf_path
+from spyder.utils.programs import find_git, run_program
```

Comment:
Is it not possible to place these imports at the top of the file? If there's no technical reason that prevents it, I think they should be left there to follow our regular convention for imports.